README.md

```diff
@@ -20,15 +20,200 @@ tasks specified in the Gruntfile.
+```
+$ echo $PATH
+  /usr/local/bin:/usr/local/sbin:/usr/bin:/bin
+```
+This means that when you type:
+```
+$ adb --version
+```
+the system will look for the command called **adb** in the directories `/usr/local/bin/`, `/usr/local/sbin/`, `/usr/bin/`, and `/bin/`.
+Note the location where you downloaded the Android SDK. It should contain a folder called `platform-tools`, which itself contains the program **adb**. If this was in the folder `/Users/someuser/Desktop/android-sdk/` you should be able to run:
+```
+$ /Users/someuser/Desktop/android-sdk/platform-tools/adb --version
+```
+This works because we're telling the computer exactly where the program **adb** exists. By putting the `platform-tools` directory on the system's *PATH* variable, we will be able to just type **adb** and have the system find it in the `/Users/someuser/Desktop/android-sdk/platform-tools/` directory.
+This process is more involved on Mac/Unix than on Windows. Use a text editor (not **Word**, but something like **TextEdit**), select the option to open a file, and browse to your home directory. You can find your home directory by typing:
+```
+$ echo ~
+```
+in a **terminal**. ('~' is a shortcut for the home directory.) Macs use a hidden file called `.bash_profile` in the home directory to set variables like *PATH*. Other Unix systems use files like `.bashrc`. You might have to check the specifics for your distribution to know which you should use. Open the appropriate file. If the file does not already exist, create a new file that will be saved with the appropriate name in your home directory.
+We want to add the location of the **adb** tool to your *PATH* while preserving the existing *PATH* information. Assuming that your **adb** program is in the `/Users/someuser/Desktop/android-sdk/platform-tools/` directory, you would add the following command to the end of the `.bash_profile` file:
+```
+$ export PATH=${PATH}:/Users/someuser/Desktop/android-sdk/platform-tools
+```
+Save the file, close the **terminal** window, open a new **terminal** window, and type:
+```
+$ echo $PATH
+```
+You should see your old path with the new directory you added above, and you should now be able to run:
+```
+$ adb --version
+```
+
+**Tip:** If you are going to be heavily customizing the look-and-feel of the application with a lot of external JavaScript libraries, you might also choose to install **bower**.
+
+## Installing Application Designer
+Open a **Terminal** and type
+```
+$ git clone https://github.com/odk-x/app-designer.git
+```
+If you're using **Github Desktop**, clone [this repository](https://github.com/odk-x/app-designer) by clicking on `Code`.
+
+On Windows, open a cmd window (go to Start Menu, search for cmd and open it); on MacOSX, open a terminal window. Within this window, type:
```

Comment:
Maybe link to the grunt section above? So you don't need to duplicate the content.